polkadot-parachain/src/chain_spec/default.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Default parachain ChainSpec based on testing rococo parachain runtime
+
```

Comment:
I wonder what this means, can we explain more here?
Why defaults based on some specific chain? Should not be other around? 

Is it 'on testing rococo' or on 'test rococo'. If second, probably test can be even omitted.

Reply:
@muharem 
Thank you for your comments, I was not sure about default.rs, but after your comments I realized and just changed `default.rs` to `rococo_parachain.rs` and now it is align with others and also it mirrors `parachains/runtimes`, and makes much more sense,
so the "Default" word just stays in `command.rs - enum Runtime`